laterpay/application/Controller/Frontend/Shortcode.php

```diff
@@ -440,13 +438,12 @@ public function render_time_passes_widget( $atts ) {
 
         $view_args = array(
             'id'                  => $data['id'],
-            'variant'             => $data['variant'],
             'introductory_text'   => $data['introductory_text'],
             'call_to_action_text' => $data['call_to_action_text'],
         );
         $this->assign( 'laterpay', $view_args );
 
-        return $this->get_text_view( 'frontend/partials/post/pass/passes' );
+        return $this->get_text_view( 'frontend/partials/post/time_pass/time_pass' );
```

Comment:
@AliaksandrVahura-ScienceSoft  please fix file name and check if we have similar issues